src/project/index.js

```diff
@@ -0,0 +1,92 @@
+window.c.project.Index = (function(m, c){
+  return {
+
+    controller: function() {
+      var vm = {recommendedCollection: m.prop([]), recentCollection: m.prop([]), expiringCollection: m.prop([])},
+          expiring = m.postgrest.filtersVM({origin: 'eq'});
+          expiring.origin('expiring');
+          recommended = m.postgrest.filtersVM({origin: 'eq'});
+          recommended.origin('recommended');
+          recents = m.postgrest.filtersVM({origin: 'eq'});
+          recents.origin('recents');
+
+      c.models.projectsForHome.getPage(1, recommended.parameters()).then(function(data){
+        vm.recommendedCollection(data);
+      });
+
+      c.models.projectsForHome.getPage(1, recents.parameters()).then(function(data){
+        vm.recentCollection(data);
+      });
+
+      c.models.projectsForHome.getPage(1, expiring.parameters()).then(function(data){
+        vm.expiringCollection(data);
+      });
+
+      return {
+        vm: vm
+      };
+    },
+
+    view: function(ctrl) {
+      var recentCollection = ctrl.vm.recentCollection,
+          recommendedCollection = ctrl.vm.recommendedCollection,
+          expiringCollection = ctrl.vm.expiringCollection;
+      return [
+        m(".w-section.section.u-marginbottom-40", [
```

Comment:
It looks like those project rows describe a component by themselves. They share a label, a common "Ver Todos" action and a predetermined number of project cards. Perhaps this page would only glue together some project-rows components to some view models. Something like: